bee-message/src/output/alias.rs

```diff
@@ -153,23 +155,31 @@ impl AliasOutputBuilder {
 pub(crate) type StateMetadataLength = BoundedU16<0, { AliasOutput::STATE_METADATA_LENGTH_MAX }>;
 
 /// Describes an alias account in the ledger that can be controlled by the state and governance controllers.
-#[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(ByteCost, Clone, Debug, Eq, PartialEq, Ord, PartialOrd)]
 #[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize))]
 pub struct AliasOutput {
     // Amount of IOTA tokens held by the output.
+    #[byte_cost(data)]
```

Comment:
Isn't there a good way to make this a default, so it doesn't need to be added to every field?

Reply:
I agree with Thoralf here, looks very redundant. Maybe make it default and create an attribute to ignore a field ?

Reply:
That's a good idea. It evolved that way because there used to be more granularity in earlier versions of the TIP.

Reply:
Interesting! So the first case would be an attribute macro (without the derive), while the second case would be a derive macro? Can they coexist in that way?